src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -281,9 +281,71 @@ def dcos_install_cli(install_location=None, client_version='1.8'):
         raise CLIError('Connection error while attempting to download client ({})'.format(err))
 
 
+def k8s_install_draft(client_version='v0.5.1', install_location=None):
+    base_url = 'https://github.com/Azure/draft/releases/download/{}/draft-{}-{}-amd64.tar.gz'
```

Comment:
This is not a good place to store a file which will be public consumed. The github release will throttle the download. Visual Studio Code once run into this issue when they pointed an extension download directly to a github release download link.

Use Azure storage blob to stage this file.

Reply:
changed draft. @seanknox sounds great, if you send me a URL I'll update (I also granted you access...)